### DISCUSSION TITLE
How to set Diff pane to display by default

### DISCUSSION THREAD

**Discussion by JN-LT** [WinMerge/winmerge#661] [Q&A] [answered]
When I compare files in WinMerge, I want the Diff pane to be displayed. This used to be the default, but lately the diff pane is always off. I have to go to the View menu and select Diff Pane every time I start WinMerge. I can't find anything in the manual or in Help that tells how to set the default or startup value so that the Diff Pane is displayed. How can I configure WinMerge such that the diff pane is displayed unless I specifically turn it off?

**Comment by sdottaka** [comment, marked as answer]
It's similar to the following bug.

https://github.com/WinMerge/winmerge/issues/138

Which version are you using?

If you are using an older version, please use version 2.16.10.

And if that doesn't help, what if I rename the following registry key in the Registry Editor?

HKEY_CURRENT_USER\SOFTWARE\Thingamahoochie\WinMerge

**Comment by JN-LT** [comment]
Perfect! I was using version 2.16.2. Upgrading to 2.16.10 fixed my problem.  I have my Diff Pane back!

Thank you very much! You’ve made my day!!

Jeff Nygren
Software Engineer II | ext. 65440
